spec/features/admin_can_edit_categories_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.